kenchen-azure-rest/swagger/Dns.swagger.json

```diff
@@ -0,0 +1,1282 @@
+            "MethodNotAllowed",
+            "NotAcceptable",
+            "ProxyAuthenticationRequired",
+            "RequestTimeout",
+            "Conflict",
+            "Gone",
+            "LengthRequired",
+            "PreconditionFailed",
+            "RequestEntityTooLarge",
+            "RequestUriTooLong",
+            "UnsupportedMediaType",
+            "RequestedRangeNotSatisfiable",
+            "ExpectationFailed",
+            "UpgradeRequired",
+            "InternalServerError",
+            "NotImplemented",
+            "BadGateway",
+            "ServiceUnavailable",
+            "GatewayTimeout",
+            "HttpVersionNotSupported"
+          ],
+          "x-ms-enum": {
+            "name": "HtpStatusCode",
+            "modelAsString": false
+          }
+        },
+        "requestId": {
+          "type": "string"
+        }
+      },
+      "description": "The response to a Zone Delete operation."
+    },
+    "ZoneListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Zone"
+          },
+          "description": "Gets or sets information about the zones in the response."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets or sets the continuation token for the next page."
+        }
+      },
+      "description": "The response to a Zone List or ListAll operation."
+    },
+    "Resource": {
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "x-ms-external": false
+    },
+    "SubResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource Id"
+        }
+      },
+      "x-ms-external": true
+    },
+    "CloudError": {
+      "type": "object",
+      "properties": { "error": { "$ref": "#/definitions/CloudErrorBody" } },
+      "x-ms-external": true
+    },
+    "CloudErrorBody": {
+      "type": "object",
+      "properties": {
+        "code": { "type": "string" },
+        "message": { "type": "string" },
+        "target": { "type": "string" },
+        "details": {
+          "type": "array",
+          "items": { "$ref": "#/definitions/CloudErrorBody" }
```

Comment:
This is the restriction of Json .Net, we're still investigating on it.

---
In reply to: [65878955](https://github.com/chenkennt/azure-rest/pull/1#discussion_r65878955) [](ancestors = 65878955)